src/libs/actions/PersonalDetails.js

```diff
@@ -183,6 +175,14 @@ function getFromReportParticipants(reports) {
 // When the app reconnects from being offline, fetch all of the personal details
 NetworkConnection.onReconnect(fetch);
 
+// Refresh the personal details every 30 minutes because there is no
+// pusher event that sends updated personal details data yet
+// See https://github.com/Expensify/ReactNativeChat/issues/468
+setInterval(fetch, 1000 * 60 * 30);
+
+// Refresh the timezone every 30 minutes
+setInterval(fetchTimezone, 1000 * 60 * 30);
```

Comment:
Oh actually. I wonder if using the interval is a good idea here in the case we are offline. I think not...

e.g.

1. You are offline
2. Every 30 minutes this will queue a new timezone fetch and personal details fetch
3. It will be placed into the network queue but not run since you are offline
4. Come back online and all the network requests will run at once.

Reply:
Said another way, there is no de-duplication for the network queue calling the same command it's fairly basic at the moment.

Reply:
Since we will fetch these on reconnect anyway... maybe we should only call `fetch` + `fetchTimezone` from these polling intervals if we are online.

Reply:
Sounds good, adding that check